pages/common/git-switch.md

```diff
@@ -12,14 +12,14 @@
 
 `git switch --create {{branch_name}}`
 
-- Create a new branch based on an existing commit:
+- Create a new branch based on an existing commit and switch to it:
```

Comment:
Maybe add a comma after "commit", to make the sentence flow a bit better?

Reply:
Hrm, that sound  odd to me. I'll add it though if you think it's a good idea.

Reply:
Sure, let's see what others think.

Reply:
Not a grammar person, but I think it is fine without the comma.

Reply:
It is fine :) I meant it as a minor suggestion, not as a correction. Since both of you prefer the no-comma version, let's go with that then. 👍